### PR TITLE
Remove more free calls

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This removes some more calls to free() so that we can be more efficient

ping @eileenmcnaughton @colemanw

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

File: CRM/Core/Payment.php [hunk 1]
```diff
@@ -261,7 +261,15 @@ public static function logPaymentNotification($params) {
     }
 
     $log = new CRM_Utils_SystemLogger();
-    $log->alert($message, $_REQUEST);
```
Comment (eileenmcnaughton) [inline CRM/Core/Payment.php]: @seamuslee001 can we put this separately

Comment (seamuslee001) [reply CRM/Core/Payment.php]: @eileenmcnaughton yes accidentally committed because I was also testing out a pr of Jon’s

**Comment by colemanw** [comment]
## No free calls!

image

File: CRM/Core/Payment.php (hunk 1 above)
Comment (seamuslee001) [reply CRM/Core/Payment.php]: Done

**Comment by eileenmcnaughton** [comment]
@seamuslee001 rebase...

**Comment by seamuslee001** [comment]
@eileenmcnaughton done

**Comment by eileenmcnaughton** [comment]
test this please

**Pull request merged**